### ISSUE TITLE
web drop-in Pre-Authorize Payment cannot input card number, expiration time, or CVC form fields

### ISSUE THREAD

**Issue by memoryoverflow** [Adyen/adyen-web#2678]
Here are some of my configurations and payment information:

I have confirmed the basic data:
  1. The client_key is correct;
        clientKey: test_YZ6FPZRLMJBHLM4K7Q5NZDP4LEK32J23
  3. The domain name is also correct;
![image](https://github.com/Adyen/adyen-web/assets/39236910/eb1b97d5-792a-4125-a308-516c67dda0bf)

Drop in test address, you can directly access and use the data provided below to reproduce the problem

china link:  http://aliyun.thisforyou.cn:180/#/

我是使用该接口进行创建支付信息的：v70版本   `/sessions`;

api request params:
```
{
    "amount": {
        "currency": "USD",
        "value": 0
    },
    "channel": "Web",
    "countryCode": "US",
    "expiresAt": 1715392937600,
    "merchantAccount": "MtpileLtdCOM",
    "recurringProcessingModel": "UnscheduledCardOnFile",
    "reference": "1788862112324632577",
    "returnUrl": "http://1788862112324632578",
    "shopperInteraction": "Ecommerce",
    "shopperLocale": "en-US",
    "shopperReference": "1788862112324632576",
    "splitCardFundingSources": false,
    "storePaymentMethod": true,
    "storePaymentMethodMode": "enabled",
    "threeDSAuthenticationOnly": false
}

```

api response:

```
{
    "amount": {
        "currency": "USD",
        "value": 0
    },
    "channel": "Web",
    "countryCode": "US",
    "expiresAt": 1715392937000,
    "id": "CS234B8404CEA90631",
    "merchantAccount": "MtpileLtdCOM",
    "mode": "embedded",
    "recurringProcessingModel": "UnscheduledCardOnFile",
    "reference": "1788862112324632577",
    "returnUrl": "http://1788862112324632578",
    "sessionData": "Ab02b4c0!BQABAgB06HMCZla+Fo8QHphBfIgC8/w1VNfBGrEJ6qc2+HNqtOj6NUTwByGYDb+M8Ra36TZ2bRdPE1i0sow/ac8rVnPpF8d2AOIHs8wak3CP9gxXzKj7IoVpWVwFUPILGZXE47VezOJEa/sH1ys/rdrl252b5c7eYUJlnhArXI5hc//n1FNAc3rqUaWHQXu70pkhQ1BgTLO6ocVpZPJ9DNxDgtEBcKLQC2QnPFLWBX1gWLKrq57wjAUEo/wwRJPKf/JnhxHEGpGIxnWefQBzqzGsLOjaX8ALppgodyU0itcjh4Mxjp9f6AUoNjjbV9h54CNl+BCe6LB+d/kXBMjyRRpyy52uEssxnEKYgkGJpLgvWrsDocvmSs4sjlPD9UBKZ7UX0vDZc7TjfxI/77D6wME+WjFe1S8n0lJHmQPsdJrVsakJQUGErEWK4aenb0IvM3DJYtrAZKsxU3cr70G0O67s10uc7wHKCPO8SAhRrX47wOBWJPgP8dhCx/9J/nIDH8DadzIclG6zNB56GSuTWZnUUbs0D06QeFz0EZKQ9PEkONYYM5YwqiEUM9hiio+ap6tmX1thkKfhTNNh4u1Bhx+QvbXZQKtzSjvbQFhClZUVthAE5/QJYHMFSPnhcJ26t7198HLhqLO8WV7hx7XTIE0JLtrqGSCx6A5+jlsilasAhgJN5Qz5ZMcEMZ3qP0g5AoAASnsia2V5IjoiQUYwQUFBMTAzQ0E1MzdFQUVEODdDMjRERDUzOTA5QjgwQTc4QTkyM0UzODIzRDY4REFDQzk0QjlGRjgzMDVEQyJ9hTrrXoHxPxIMv6+eaqC7dHZTH8si9aqg8NwmprOPYrA3Lf7XZXVQMDnXpjYhf/SCkzX5NSaLXKTGycjdN7E6E+ItOvPBXbHieOqwHaqaSse53wWDj7NKKNuKXzhnjbeNnYW/5nScGefXGA2gAkpFg9MYjg7D4owS48jUcwF7CpDftg4+RuH+d8StI8wpr5iyZz2kVl0lyicZkXs0SaJOhQGcM/K+1e1SAhA2y5NCphHrws9z6EJstBTmwExOQwogtUsKUw1HFM0z9HQM7KnquUtTjU5PrmN3Ih4nzgf6OyxLTTKeW5+tpc92nuB6kJ8wBqMpnP18a+/vuUlUTRsWn5Xp6I9izmUQLC9N6zmDWCHsQQmm9z6485obnnJ4JLhin1JPYThhxogo8ZFWXtFJ3iN7cPwjYCIwDEOcf78deLWsamYotSOqqMnZhBkj5RjNfOGxtzr+CWm4IQ6fOnMZDdjDKt9b9kGYnL0eDdI6wJPmZsbLYAE40BeKisKcz1OY5E8CGJl+6HlIrYCvCLpVv09lufckBDcnTt3/e/V2XUll63uhJSmPhL4a1VtTxmbV7xWdgcRAyoAPGQd5Ai3liOu/8Zqv64XxPc2L/Oq60B+AM+EmAhs+ES/nkgoRUvG3OSazHTBMixYRZlSXC4LGiEb45rXEimiFQeS8imnd1ogU+xxS92o9vGmBqo8uh3zM4YihSI+eVQmb13VbdCru7ED1YjM+PHKeBnpCRtDP2y7z7Wpch7f0zd8Vs5aaTerkQ0qoOZ1MLtoVR6J9m7s5NP1if9UNLkLRMzNTZhn5WMJwOWktL+q5sI27+f8+zp27eSnuAjuSz72l",
    "shopperInteraction": "Ecommerce",
    "shopperLocale": "en-US",
    "shopperReference": "1788862112324632576",
    "splitCardFundingSources": false,
    "storePaymentMethod": true,
    "storePaymentMethodMode": "enabled",
    "threeDSAuthenticationOnly": false
}
```

Drop-in effect：

![image](https://github.com/Adyen/adyen-web/assets/39236910/78047f2d-c0c0-41e3-9edf-68ca2ad602db)

web console:
![image](https://github.com/Adyen/adyen-web/assets/39236910/99874c85-5f03-421b-9462-76aad265bd35)

**Comment by longyulongyu** [comment]
Hi @memoryoverflow I think for the allowed origins, the protocol should be https, or http for local, you can find more information [here](https://docs.adyen.com/development-resources/client-side-authentication/#allowed-origins).

**Comment by sponglord** [comment]
@memoryoverflow - your issue relates to this: https://github.com/Adyen/adyen-web/issues/1710 
TL;DR `http` is not considered a _secure context_ unless it is `localhost`

**Comment by memoryoverflow** [comment]
Thank you for your reply. Now you can initiate payment normally. I think it should be caused by the mismatch between the client key and the domain name.

**Comment by longyulongyu** [comment]
> Thank you for your reply. Now you can initiate payment normally. I think it should be caused by the mismatch between the client key and the domain name.

@memoryoverflo good to hear that you can initiate payment normally now.
Is the issue solved by matching the client key and domain name? Did you also change the allowed origins as we mentioned?
